homeassistant/components/aquostv/media_player.py

```diff
@@ -158,13 +158,19 @@ def turn_off(self) -> None:
         self._remote.power(0)
 
     @_retry
-    def volume_up(self):
+    def volume_up(self) -> None:
         """Volume up the media player."""
+        if self.volume_level is None:
```

Comment:
`mypy` didn't like the previous implementation because `self.volume_level` can be `None`
I thought of using `assert` but I think this is nicer.